test/Elastica/Base.php

```diff
@@ -123,7 +123,7 @@ protected function _createIndex($name = null, $delete = true, $shards = 1)
 
         $client = $this->_getClient();
         $index = $client->getIndex('elastica_'.$name);
-        $index->create(['index' => ['number_of_shards' => $shards, 'number_of_replicas' => 0]], $delete);
+        $index->create(['index' => ['number_of_shards' => $shards, 'number_of_replicas' => 2]], $delete);
```

Comment:
As you have only 2 instances of elasticsearch, this should probably be set to 1 and not 2?